test/framework/clusterctl_helper.go

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
do we want a build tag here?
```suggestion
//go:build e2e
// +build e2e

/*
```